fluent-package/msi/source.wxs

```diff
@@ -228,13 +221,38 @@
                   Return="check"
                   Impersonate="no" />
 
+    <!--
+        DO NOT DELETE FLUENT_PACKAGE_TOPDIR
+        It must be set because during msi installation process,
+        InstallFluentdWinSvc set path of fluentd.conf via --reg-winsvc-fluentdopt then
+        fluent/winsvc.rb spawn ruby process directly as a service.
+        Thus FLUENT_PACKAGE_TOPDIR must be set explicitly as an
+        environment variable.
+        Usually, environment variable set via Environment element as Component,
+        but directly separator must be changed to '/', so we need to use CustomAction.
+    -->
+    <CustomAction Id="ModifyDirSepForFluentPackageTopdir"
+                  Script="vbscript">
+      <![CDATA[
+        pathvalue = Session.Property("FLUENTPROJECTLOCATION")
+        newValue = Replace(pathvalue, "\\", "/")
+        Session.Property("FLUENTPROJECTLOCATION") = newValue
+        shellObject = CreateObject("WScript.Shell")
+        systemEnv = objShell.Environment("SYSTEM")
+        systemEnv("FLUENT_PACKAGE_TOPDIR") = newValue
+      ]]>
+    </CustomAction>
+    <!-- Propagate changed environment variable -->
+    <CustomActionRef Id="WixBroadcastEnvironmentChange" />
```

Comment:
Wow! I had no idea there was such a way!

I'm concerned about one point.
Don't we need to remove the environmental variable on uninstalling?

Reply:
It is just a PoC yet because now building msi...

If it works as expected, also add code for uninstalling.

Reply:
I see!

Reply:
Failed.  :sob: 

```
MSI (s) (F8:7C) [16:28:25:209]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1720 
Error 1720. There is a problem with this Windows Installer package. A script required for this install to complete could not be run. Contact your support personnel or package vendor.  Custom action ModifyDirSepForFluentPackageTopdir script error -2146827850, Microsoft VBScript 実行時エラー: オブジェクトでサポート
されていないプロパティまたはメソッドです。: 'shellObject' Line 4, Column 9,  
MSI (s) (F8:7C) [16:28:53:263]: Note: 1: 2205 2:  3: Error 
MSI (s) (F8:7C) [16:28:53:263]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1709 
MSI (s) (F8:7C) [16:28:53:263]: Product: Fluent Package v5.0.0 -- Error 1720. There is a problem with this Windows Installer package. A script required for this install to complete could not be run. Contact your support personnel or package vendor.  Custom action ModifyDirSepForFluentPackageTopdir script error -2146827850, Microsoft VBScript 実行時エラー: オブジェクトでサポートされていないプロパティまたはメソッドです。: 'shellObject' Line 4, Column 9,  
```

Reply:
investigating a workaround...

Reply:
Got it.

It should be `Session.Property("FLUENTPROJECTLOCATRION").

Reply:
we can choose inline snippet or external script.

Reply:
rebuilding msi...

Reply:
It should set Impersonate="no" explicitly.

Reply:
There are two way to do it.

* Kick inline script via CustomAction
* Kick external script via CustomAction with JScriptCall and so on.

Both of them, it seems that Session.Property("...") is empty.
Additionally, setting Session.Property can't be overridden.

Reply:
> Both of them, it seems that Session.Property("...") is empty.

This is a blocker of this issue.